curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-317-firecracker.md

```diff
@@ -8,31 +8,31 @@ dashedName: problem-317-firecracker
 
 # --description--
 
-A firecracker explodes at a height of 100 m above level ground. It breaks into a large number of very small fragments, which move in every direction; all of them have the same initial velocity of 20 m/s.
+A firecracker explodes at a height of 100 m above level ground. It breaks into a large number of very small fragments, which move in every direction; all of them have the same initial velocity of 20 $\frac{\text{m}}{\text{s}}$.
 
-We assume that the fragments move without air resistance, in a uniform gravitational field with g=9.81 m/s2.
+We assume that the fragments move without air resistance, in a uniform gravitational field with $g=9.81 \frac{\text{m}}{\text{s}^2}$.
```

Comment:
I'm not sure if this one is supposed to be `m/(s^2)` or `(m/s)^2`. I feel like it's the latter, but I'm not certain. Here's the [Euler page](https://projecteuler.net/problem=317). I added a suggestion to make it the latter, but only if you agree @gikf.

```suggestion
We assume that the fragments move without air resistance, in a uniform gravitational field with $g=9.81 \frac{\text{m}}{\text{s}}^2$.
```

Reply:
This one is definitely m/(s^2), as that's acceleration https://en.wikipedia.org/wiki/Gravitational_acceleration